lib/Wikia/tests/Service/Helios/HeliosClientTest.php

```diff
@@ -7,25 +7,22 @@
 
 class HeliosClientTest extends \WikiaBaseTest {
 
-	public function setUp()
-	{
+	public function setUp() {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this method "setUp" to simply inherit it. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1185)